gips/0055-chain-integration-process.md

```diff
@@ -0,0 +1,97 @@
+
+### Stage 3
+
+
+The last stage is supporting the new blockchain on mainnet with some level of support for arbitration and indexing rewards. For this, integrators must:
+1. Open a PR,updating the `[feature-matrix-support.md](http://feature-matrix-support.md)` [here](https://github.com/graphprotocol/indexer/blob/main/docs/feature-support-matrix.md) to include support for the new data source.
+2. Publish a GIP [template here](https://github.com/graphprotocol/graph-improvement-proposals/blob/main/gips/0000-template.md) as the formal process to request Graph protocol integration. The GIP must include the proposed update to the Feature Matrix Support (the open PR) and data gathered during Stage 2 on performance, stability, and data determinism.
+
+
+Afterward, the Graph Foundation will post the GGP (Graph Governance Proposal) on the [Graph’s Council snapshot page](https://snapshot.org/#/council.graphprotocol.eth). The GGP will follow the principles agreed upon in [GIP-008: Subgraph API Versioning and Feature Support](https://github.com/graphprotocol/graph-improvement-proposals/blob/main/gips/0008-subgraph-api-versioning-and-feature-support.md). The Graph Foundation will ensure the GGP is created and reviewed by the Graph Council. Council's ratification of the GGP will officially mark the support for the new data source on The Graph's mainnet as outlined in the Feature Matrix Support (GIP).
+
+> *Note*: The GGP vote may be preceded by a community poll, providing no rough consensus on readiness and value-added for the protocol. Core developers’ and Indexers’ confidence in the integration stability, performance, reliability, and overall demand (the focus of Stage 2) will provide the Council with the required information to proceed with the GGP vote.
+
+## Summarized Requirements & Exit Criteria
+
+| | Requirements | Exit Criteria |
+| --- | --- | --- |
+| Stage 1 |✓ A [new Forum topic](https://forum.thegraph.com/c/governance-gips/new-chain-support/71) if Graph Node integration exists; GRP if otherwise (for one to be built).<br>✓ Open PRs in [Graph Node](https://github.com/graphprotocol/graph-node), [graph-ts](https://github.com/graphprotocol/graph-tooling/tree/main/packages/ts) and [graph-cli](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli) repositories. More info [here](https://github.com/graphprotocol/graph-node/blob/master/docs/implementation/add-chain.md).<br>✓ Docker and bare metal Indexer guides for Firehose stack and EVM RPC.`ᵇ`<br>✓ Example subgraph with docs.`ᶜ` <br> ✓ Open-sourced Firehose implementation, if applicable. |→ Chain's node and/or Firehose runs reliably.`ᵃ`.<br> → Example subgraph syncs successfully.|
```

Comment:
Graph Node and Graph TS PRs should specify "if applicable"
Docker and bare metal Indexer guides for Firehose stack and EVM RPC should specify "as appropriate"